gradle-language-server/src/main/java/com/microsoft/gradle/resolver/GradleLibraryResolver.java

```diff
@@ -147,12 +158,18 @@ public void loadGradleClasses() {
 		}
 	}
 
-	private DistInfo getWrapperPropertiesInfo() {
-		if (this.workspacePath == null) {
+	private DistInfo getWrapperPropertiesInfo(URI relativeToGradleFile) {
```

Comment:
When I was debugging the change, I found that the value of `relativeToGradleFile` is an absolute path. Is the parameter called with `relative` for some reasons?

Reply:
The reason to call it `relative*` was the path computed was based on the gradle file we find while traversing upwards. I could just call it may be like `gradleModulePath`. Let me know if you have a better suggestion.

Reply:
I saw the `URI` is directly get from the request of completion.

![image](https://github.com/microsoft/vscode-gradle/assets/6193897/3049a619-86b7-4810-ac95-c4cb862fa642)

I didn't see things about `traversing upwards`. Do I miss anything?

Reply:
You are correct, i mixed up with a spotless plugin change i looked into 🙂. I think we can change this to `projectPath`, WDYT?

Reply:
What about `gradleFilePath`?

Reply:
Yes that sounds good. I will update the PR